lab-james/server.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const http = require('http');
+const Game = require('./model/game.js');
+const Router = require('./lib/router.js');
+const storage = require('./lib/storage.js');
+const PORT = process.env.PORT || 3000;
+
+const router = new Router();
+
+const server = module.exports = http.createServer(router.route());
+server.listen(PORT, () => console.log(`Listening on ${PORT}`));
+
```

Comment:
keep building out your routes! I'd recommend starting with your POST route so you can have an instance of your resource to .GET, .PUT(update), and .delete

hint: you'll be using your storage module methods inside of them just like your .get